mmselfsup/models/backbones/cae_pretrain_vit.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Better to name `cae_pretrain_vit.py` to `cae_vit.py`. `mae_pretrain_vit.py`should also be renamed to `mae_vit.py`.